db/migrate/20190430212446_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :books do |t|
+      t.string :genre
```

Comment:
Remember [Null modifier](https://edgeguides.rubyonrails.org/active_record_migrations.html#column-modifiers)

Reply:
@fedecavazzoli review this, please.